load_balancers.go

```diff
@@ -90,12 +95,12 @@ func (l LoadBalancer) AsRequest() *LoadBalancerRequest {
 		RedirectHttpToHttps:          l.RedirectHttpToHttps,
 		EnableProxyProtocol:          l.EnableProxyProtocol,
 		EnableBackendKeepalive:       l.EnableBackendKeepalive,
-		HealthCheck:                  l.HealthCheck,
```

Comment:
Why did you delete this?

Reply:
There's already another logic in place to set the health check params below
https://github.com/digitalocean/godo/blob/b05b66a25464c7b517efc07f1f20095cce510518/load_balancers.go#L110-L113

Reply:
This isn't a breaking change, right?

Reply:
Shouldn't be a breaking change: we're simply removing a redundant assignment statement.